src/ObjectCollection/DateTime/ContainsDateTimeStrictValueTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Steevanb\PhpCollection\ObjectCollection\DateTime;
+
+trait ContainsDateTimeStrictValueTrait
```

Comment:
j'ai pas précisé ailleurs dans la lib quand on n'a que le typage demandé, c'est quand on autorise null que j'ai précisé
```suggestion
trait ContainsDateTimeValueTrait
```